wercker.yml

```diff
@@ -0,0 +1,29 @@
+box: blisteringherb/scholarship@0.0.6
```

Comment:
Yeah, I should've named it differently a long time ago, probably something like dosomething-laravel, but it's a basic box to build laravel apps on.